new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+[«<Esc>»](<Esc>) and then [u](u) will undo the latest change.
+
+This tutorial is interactive, and there are a few things you should know.
+Pressing [«<Enter>»](<Enter>) over text highlighted [like this](holy-grail)
+will take you to some relevant help (hopefully), and pressing ? over any word
+will try to do so too.  You can close the help window by pressing «q».
+Sometimes you will be required to modify text like
+
+--->  this here {expect:this here}
+
+Once you have done the changes correctly, the ✗ sign at the left will change to
+✓. We imagine you can already see how neat Vim can be ;) Other times, you'll be
+prompted to run a command (we will explain this later):
+
+~~~ viml
+    :help <Enter>
+~~~
+
+or press a sequence of keys
+
+    <Esc>0f<SPACE>d3wP$P
+
+Text within angle brackets (like <Enter>) describes a key to press instead of
+text to type.
+
+Now, make sure that your Shift-Lock key is NOT depressed and press the «j» key
+enough times to move the cursor so that Lesson 1.1 completely fills the screen.
+
+## Lesson 1.1: MOVING THE CURSOR
+
+** To move the cursor, press the «h», «j», «k», «l» keys as indicated. **
+
+        ← ↓ ↑ →    Hint: The h key is at the left and moves left.
+        h j k l          The l key is at the right and moves right.
+                         The j key looks like a down arrow.
+
+
+  1. Move the cursor around the screen until you are comfortable.
+
+  2. Hold down the «j» and move to the next lesson.
+
+
+NOTE: If you are ever unsure about something you typed, press <Esc> to place
+      you in Normal mode.  Then retype the command you wanted.
+
+NOTE: The cursor keys should also work, but the «h» «j» «k» «l» keys have not
+      been chosen arbitrarily, they are on the right-hand side of the home row,
+      allowing you to mode the curse without taking youre hand off. Once
+      getting used to «h» «j» «k» «l» you will be able to move around much
+      faster. Really!
+
+# Lesson 1.2: EXITING VIM
+
+!! NOTE: Before executing any of the steps below, read this entire lesson !!
+
+  1. Press the <Esc> key (to make sure you are in [Normal](Normal) [mode](vim-modes)).
+
+  2. Type:
+
+     :q! <Enter>.
+
+     This exits the editor, DISCARDING any changes you have made. The
+     exclamation mark "supercharges" the [:q](:q) command.
+
+  3. Open vim and get back here by executing the command that got you into this
+     tutor. That might be:
+
+     :Vimtutor <Enter>
+
+  4. If you have these steps memorized and are confident, execute steps
+     1 through 3 to exit and re-enter the editor.
+
+NOTE:  The command  [:q!](:q) <Enter>  discards any changes you have made. In a
+       few lessons you will learn how to save the changes to a file.
+
+  5. Move the cursor down to Lesson 1.3.
+
+# Part 1: COUNT, OPERATOR, MOTION -- ACTION
+
+Before we continue we need to understand the concept of modal editing first:
+most editors have two "modes", a mode where you enter text and a mode where you
+select text.
```

Comment:
Does it really matter what device they use? I assume anyone interested in the power of Vim is already aware of the power of using Shift and the arrow keys to select text quickly and is interested in an even more effective way of editing text.

I think a better clarification would be `where you select text visually`.